packages/hoppscotch-sh-admin/src/components/app/Sidebar.vue

```diff
@@ -67,12 +80,20 @@ import IconUser from '~icons/lucide/user';
 import IconUsers from '~icons/lucide/users';
 import IconSettings from '~icons/lucide/settings';
 import { useI18n } from '~/composables/i18n';
+import { ref } from 'vue';
 
 const t = useI18n();
 
 const { isOpen, isExpanded } = useSidebar();
 
-const primaryNavigations = [
+type NavigationItem = {
+  label: string;
+  icon: any;
```

Comment:
We can use `Component` imported from `vue` here instead.

```suggestion
  icon: Component;
```